doc/source/whatsnew/v0.19.1.txt

```diff
@@ -25,6 +25,13 @@ Performance Improvements
 
 
 
+.. _whatsnew_0191.new_features:
+
+New features
+~~~~~~~~~~~~
+- Add checks to assure that left_index and right_index are of type bool
```

Comment:
Can you move this to the bug fixes section? (I think that it is rather a bug that it accepted a wrong value, than a new feature that it now checks that)